src/nav-soknad/components/avbrytsoknad/AvbrytSoknad.tsx

```diff
@@ -19,7 +21,15 @@ export const AvbrytSoknad = () => {
 
     const onAvbryt = () => {
```

Comment:
onAvbryt høres litt feil ut for meg. Mulig jeg missforstår.

Reply:
Er ganske så enig. Tenker å ta en egen runde etterhvert hvor vi ser nærmere på språk, variabelnavn og sånt. :)